### ISSUE TITLE
[BUG] Runtime.ImportModuleError in every Lambda function after deploying new project 

### ISSUE THREAD

**Issue by podenborg** [flowaccount/nx-plugins#92]
**Describe the bug**
All of my Lambda functions are throwing `Runtime.ImportModuleError` errors after deploying a new project with the plugin's `nx deploy <myapi>` command.

I get errors similar to the one below:

``` json
{
    "errorType": "Runtime.ImportModuleError",
    "errorMessage": "Error: Cannot find module 'axios'",
    "stack": [
        "Runtime.ImportModuleError: Error: Cannot find module 'axios'",
        "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
        "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
        "    at Object.<anonymous> (/var/runtime/index.js:45:30)",
        "    at Module._compile (internal/modules/cjs/loader.js:778:30)",
        "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)",
        "    at Module.load (internal/modules/cjs/loader.js:653:32)",
        "    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:585:3)",
        "    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)",
        "    at startup (internal/bootstrap/node.js:283:19)"
    ]
}
```

I created a new Node-Typescript project using the plugin's generator command and have not modified the serverless.yml config. Here's my serverless config file:

``` yml
service: myapi
frameworkVersion: ">=1.1.0"
plugins:
  - serverless-offline
package:
  individually: true
  excludeDevDependencies: false
provider:
  name: aws
  region: us-east-1
  runtime: nodejs10.x
functions:
  hello-world:
    handler: src/handler.helloWorld
    events:
      - http:
          path: hello-world
          method: get
```

I haven't modified any of the packages installed by the plugin:

``` json
{
  ...
  "dependencies": {
    "axios": "0.21.1",
    ...
  },
  "devDependencies": {
    "@flowaccount/nx-serverless": "^1.0.0-beta.3",
    "serverless": "^2.29.0",
    "serverless-offline": "7.0.0",
    ...
  }
}
```

The version of the `serverless` package that it installed to my `node_modules` folder is version `2.53.1`.

**To Reproduce**

1. Generate a new node-typescript application with the plugin
2. Install any npm package (I used axios in my example)
3. Import and use said package in one of your lambda functions
4. Deploy the project to AWS
5. Invoke the deployed lambda function

**Expected behavior**
I would expect the modules to resolve properly and for the project to work after deployment in a new project

**Check which provider is affected:**
[x] AWS
[] Azure
[] Google Cloud Platform

**Check which framework is affected:**
[] Angular
[x] Nodejs
[x] Serverless
[x] Lambda
[] Infrastructure as a code

**Additional context**

**Comment by dornellas13** [comment]
hello i'm having the same problem, find out what it can be?

**Comment by wickstargazer** [comment]
does this `https://github.com/flowaccount/nx-plugins/pull/89` solves it?
just released `1.0.0-beta.5`
